### DISCUSSION TITLE
Battery backup

### DISCUSSION THREAD

**Discussion by MWright1952** [gopro/labs#998] [Q&A]
I'm having an issue w/ Suptig 7800 power backup... With my Hero 4 I connected it without the Gopro battery installed and it worked fine.  With my 11 Black, though, I got an error message that there wasn't enough power without the camera battery.  It stated that I needed 5V, 2A.  That is what the Suptig put out!  I downloaded and installed Labs thinking that TUSB=1 would solve this issue, but it doesn't.  Now removing the battery and plugging in the backup gives me nothing.  No power at all.  Ideas?

**Comment by dnewman-gpsw** [comment]
Please share the exactly command you used.  Trust USB does work, and doesn't change anything else about the camera.

**Reply by MWright1952** [reply]
TUPS=1  I created a QR code with that command, scanned it w/ my 11 Black and nothing... removing the camera battery and plugging in the external battery gives me nothing.  Camera doesn't come on and the battery shuts down.

**Reply by MWright1952** [reply]
In addition, plugging the external battery in with the Gopro battery installed and the camera doesn't charge... it's as if the camera doesn't see the external battery at all

**Reply by dnewman-gpsw** [reply]
The command is *TUSB=1, so you haven't yet used a valid command.  Sound like you have a real issue with you external power.

QR URL https://gopro.github.io/labs/control/set/?cmd=*TUSB=1

**Reply by MWright1952** [reply]
Sorry, typo... TUSB.  I used the QRcontrol App.
